lncfg/dev_integration.go

```diff
@@ -18,9 +20,29 @@ func IsDevBuild() bool {
 //nolint:lll
 type DevConfig struct {
 	ProcessChannelReadyWait time.Duration `long:"processchannelreadywait" description:"Time to sleep before processing remote node's channel_ready message."`
+	ReservationTimeout      time.Duration `long:"reservationtimeout" description:"The maximum time we keep a pending channel open flow in memory."`
+	ZombieSweeperInterval   time.Duration `long:"zombiesweeperinterval" description:"The time interval at which channel opening flows are evaluated for zombie status."`
 }
 
 // ChannelReadyWait returns the config value `ProcessChannelReadyWait`.
 func (d *DevConfig) ChannelReadyWait() time.Duration {
 	return d.ProcessChannelReadyWait
 }
+
+// GetReservationTimeout returns the config value for `ReservationTimeout`.
+func (d *DevConfig) GetReservationTimeout() time.Duration {
+	if d.ReservationTimeout == 0 {
+		return 10 * time.Minute
```

Comment:
Unfortunately I cannot use the Default specified in the lncfg package this will lead to a import cycle, maybe there is a better solution?

Reply:
I think you can just  return `DefaultReservationTimeout` here? Don't think there would be an import cycle.

Reply:
True, not sure what I did locally